homeassistant/components/myio/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""The myIO integration."""
+from datetime import timedelta
+import logging
+
+from myio.comms_thread import CommsThread  # pylint: disable=import-error
+
+from homeassistant.const import CONF_NAME
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+from homeassistant.util import slugify
+
+from .const import CONF_REFRESH_TIME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["sensor"]
+
+COMMS_THREAD = CommsThread()
+
+
+async def async_setup(hass, config):
+    """Set up the myIO-server component."""
+    return True
+
+
+async def async_setup_entry(hass, config_entry):
+    """Set up config entry."""
+    _LOGGER.debug(DOMAIN)
+    _server_name = slugify(config_entry.data[CONF_NAME])
+
+    try:
+        hass.data[DOMAIN][_server_name] = {}
+    except (KeyError):
+        hass.data[DOMAIN] = {}
```

Comment:
Put this line in async_setup() then it's always done before loading entries.

Reply:
The _server_name variable is from config_entry. 
The user can use more server at same time (one at upstairs, one at downstairs, one at the garage) ... so I have to make difference beetween the servers databases, that is the reason, why I use the _server_name key.

Reply:
The entry id will be different for each entry setup.

Reply:
the server name must be different too...